scripts/checkMissingTranslations.ts

```diff
@@ -0,0 +1,59 @@
+import { Resource, ResourceKey } from 'i18next'
+import i18n from '../src/i18n'
+
+const checkRecursiveTranslation = (
+  searchingPath: string,
+  defaultLanguageObject: ResourceKey,
+  translatedLanguageObject: ResourceKey,
+) => {
+  if (typeof defaultLanguageObject === 'string' || typeof translatedLanguageObject === 'string') {
+    if (typeof defaultLanguageObject === 'object') {
+      console.log(
+        `📙 Found a string for path ${searchingPath} while is and object for the default language`,
+      )
+    }
+    return
+  }
+  const defaultKeys: string[] = Object.keys(defaultLanguageObject)
+  const translatedKeys: string[] = Object.keys(translatedLanguageObject)
+  if (defaultKeys.length === 0 || translatedKeys.length === 0) {
+    return
+  }
+  defaultKeys.forEach((key) => {
+    if (!translatedLanguageObject[key]) {
+      console.log(`📙 The key ${key} is not present for path ${searchingPath}`)
```

Comment:
I'm thinking that we also add the language code to the log message.. something like 

`[${languageCode}]: No translation present for ${key} in ${searchPath}`

Reply:
Yes you’re right